assets/app/view/game_page.rb

```diff
@@ -216,7 +216,10 @@ def menu
 
     def item(name, anchor = '')
       change_anchor = lambda do
-        store(:scroll_pos, `document.getElementById('chatlog').scrollTop`, skip: true) unless route_anchor
+        unless route_anchor
+          elm = Native(`document.getElementById('chatlog')`)
+          store(:scroll_pos, elm.scrollTop < elm.scrollHeight - 220 ? elm.scrollTop : nil, skip: true)
```

Comment:
perhaps set 220 as a constant, so future us know why 220?

Reply:
Nope, replacing it with offsetHeight right now, because I neglected chatlog height change by user …